mark.c

```diff
@@ -807,7 +815,39 @@ GC_INNER mse * GC_mark_from(mse *mark_stack_top, mse *mark_stack,
 #     ifndef SMALL_CONFIG
         word deferred;
 
+#       if !defined(__CHERI_PURE_CAPABILITY__)
+        /* Try to prefetch the next pointer to be examined ASAP.        */
+        /* Empirically, this also seems to help slightly without        */
+        /* prefetches, at least on linux/X86.  Presumably this loop     */
+        /* ends up with less register pressure, and gcc thus ends up    */
+        /* generating slightly better code.  Overall gcc code quality   */
+        /* for this loop is still not great.                            */
+        for(;;) {
+          PREFETCH(limit - PREF_DIST*CACHE_LINE_SIZE);
+          GC_ASSERT((word)limit >= (word)current_p);
+          deferred = *(word *)limit;
+          FIXUP_POINTER(deferred);
+	  limit -= ALIGNMENT; 
```

Comment:
Indent issue?

Reply:
autoindent gone wrong. Thanks for catching that.